src/commands/send.ts

```diff
@@ -27,12 +24,14 @@ export class SendCommand extends WriteCommand {
         `$ uns send 1237.77 --to DNLmWfFkXHcrBHmr8UTWpNGmTrX9WohZH3 --network devnet --format json`,
     ];
 
+    public static DEFAULT_FEES: number = 1; // 1UNS fees (100000000 if sato flag is activated)
+
     public static flags = {
-        ...WriteCommand.flags,
+        ...WriteCommand.getWriteCommandFlags(SendCommand.DEFAULT_FEES),
         ...passphraseFlag,
         ...secondPassphraseFlag,
         to: flags.string({
-            description: "The recipient address.",
+            description: "The recipient public address OR the @unik-name of the recipient.",
             required: true,
         }),
         check: flags.boolean({
```

Comment:
`check` description is not good. It says that `--check` "allow sending tokens to an address that do not exists on chain". 
It's the opposite, if you set `--check` it won't allow sending tokens to an address that does not exist on chain.

and spelling is wrong. It's "that does not exist on chain"